config/config.php

```diff
@@ -34,20 +37,30 @@
             ],
         ],
         'statuses_labels' => [
-            'draft' => 'Draft',
-            'in_cart' => 'Cart',
-            'pending' => 'Pending',
-            'approved' => 'Approved',
-            'rejected' => 'Rejected',
-            'scheduled' => 'Scheduled',
-            'cancelled' => 'Cancelled',
-            'completed' => 'Completed',
+            S::STATUS_DRAFT => 'Draft',
```

Comment:
Got it, I will fix it. I thought that having this keys in centralized place would be better. But if we think about it like about static file that it would be better to not rely on PHP-specific things.

Reply:
I use constants in my config as well, but I also kept in mind that it should be static and decided to use PHP constructs as well. Here's my reasoning:

Any other format we might employ in the future can have a mechanism similar to the config's replacement, such as detection of constants or retrieval of values from the container (instead of string interpolation). No matter what it might end up being, there should be a way for the config to have values that area declared elsewhere, maybe in PHP code.

With that in mind, I arrived to the conclusion that the current format has such a "mechanism". Since it's PHP, we can reference constant names. But of course, this can change along with the format, and IMO the above does not pose any problems in doing so.